scripts/db_restore.sh

```diff
@@ -72,12 +74,16 @@ fi
   set -e
 } | tee >/dev/null
 
+# Update table permissions
+echo "Updating table permissions"
+psql -U postgres -h localhost $LOCALDBNAME -c "GRANT ALL ON SCHEMA public TO ${LOCALDBUSER};"
+
```

Comment:
See https://medium.com/@dmitry.romanoff/postgresql-version-15-error-permission-denied-for-schema-public-dc8285c41e00